src/app/clients/getExpandedClientDetails.ts

```diff
@@ -181,18 +182,26 @@ export const formatBreakdownOfChildrenFromFamilyDetails = (
             familyMember.birth_year >= getCurrentYear() - 16 &&
             familyMember.birth_year < getCurrentYear() - 2
         ) {
-            const age = getChildAgeUsingBirthYearAndMonth(familyMember.birth_year, false);
+            const age = getChildAgeUsingBirthYearAndMonth(
+                familyMember.birth_year,
+                familyMember.birth_month,
+                false
+            );
             childDetails.push(`${age} ${familyMember.gender}`);
         } else if (familyMember.birth_year >= getCurrentYear() - 2) {
             if (familyMember.birth_month) {
                 const age = getChildAgeUsingBirthYearAndMonth(
                     familyMember.birth_year,
-                    false,
-                    familyMember.birth_month
+                    familyMember.birth_month,
+                    false
                 );
                 childDetails.push(`${age} ${familyMember.gender}`);
             } else {
-                const age = getChildAgeUsingBirthYearAndMonth(familyMember.birth_year, false);
+                const age = getChildAgeUsingBirthYearAndMonth(
+                    familyMember.birth_year,
+                    familyMember.birth_month,
+                    false
+                );
                 childDetails.push(`${age} ${familyMember.gender}`);
             }
```

Comment:
i think the if and else do exactly the same thing, so why not just remove the if statement?